---
layout: page
title: research
permalink: /research/
description: Using computational science approaches to study ecosystem effects of aquatic microbial diversity
nav: true
nav_order: 3
horizontal: false
---

### General Project Themes
My research projects fall into one or more of three major categories:

1. **Protistan diversity and taxonomy-function relationships**
2. **Phytoplankton succession & community dynamics**
3. **Phytoplankton biogeography**

Using one or more of these tools:

1. **Experimental culturing**
2. **Models (ecosystem & physiological)**
3. **-Omics (nucleic acid; population or community)**

### Recent Project Examples

#### Physiology experiments to study multiple stressors & interactions

The vast molecular diversity of phytoplankton and other protists can be tied to large-scale processes and nutrient biogeochemistry via physiology. Physiology experiments track the rate of population growth, nutrient use, or metabolite production by isolates or communities. I design physiology experiments to link genetic diversity, for example strain differentiation in the cosmopolitan coccolithophore _Gephyrocapsa huxleyi_, to traits that models can be parameterized with, for example thermal niche width or habitat.

#### Ecology & evolution of marine microbial eukaryotes

Aquatic microbial eukaryotes have complex genomes that are difficult to reconstruct from environmental data, and individual cells can be challenging to culture in the laboratory. Using -omics approaches, we can now study the inter- and intraspecific diversity in biogeochemically-relevant strains of marine protists. My current projects on this topic include:

1. Extracting genes from novel lineages from meta-omic data using clustering approaches
2. Studying strain specificity and pangenomic variability in cosmopolitan phytoplankton taxa (_Gephyrocapsa huxleyi_)

####  Marine microeukaryotic meta-omics: tool development & ecological research

<center><p align="center" style="max-width:60%;">
  <img width="100%" height="auto" src="/assets/img/01-eukrhythmic-flowchart.png" />
</p></center>

_Metatranscriptomics_, or the measurement of the expression levels of an entire community from a single sample, has an important role to play in microbial oceanography. We often can _only get_ a single sample easily, and there's an enormous amount of uncultured diversity present in those samples. For that reason, marine metatranscriptomics have potential both to tell us about organisms that we aren't able to cultivate in the lab, and to tell us how the _expression_ of various genes changes over time. This simultaneously tells us what microbes are present in a sample, and what biogeochemical roles they might be performing.

<center><p align="center" style="max-width:60%;">
  <img width="100%" height="auto" src="/assets/img/09-tara.png" />
</p></center>

I am interested in marine microbial eukaryotes, or protists, which present the secondary challenge that a lot of -omics tools are not developed for eukaryotic organisms. For this reason, I have developed _euk_`rhythmic`, a metatranscriptomic pipeline that consolidates the output of multiple individual assemblers to produce a cohesive assembly process that can be used for downstream analysis and posing meaningful biological and oceanographic questions. This work complements ongoing software development for accurate taxonomic identification of marine eukaryotes in meta-omic samples (`EUKulele`).

<center><p align="center" style="max-width:60%;">
  <img width="100%" height="auto" src="/assets/img/08-narragansett.png" />
</p></center>